cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -129,31 +125,40 @@ mkTracers :: forall peer blk.
            -> Tracers peer blk
 mkTracers traceOptions tracer = Tracers
     { chainDBTracer
-        = annotateSeverity $ filterSeverity (pure . const (tracingSeverity $ traceChainDB traceOptions))
-          $ teeTraceChainTip (tracingFormatting $ traceChainDB traceOptions) tracingVerbosity
-          $ addName "ChainDB" tracer
+        = tracerOnOff (traceChainDB traceOptions)
+            $ annotateSeverity $ filterSeverity (pure . const (tracingSeverity $ traceChainDB traceOptions))
```

Comment:
filtering based on `Severity` (tracingSeverity) and controlling the verbosity of the output (tracingVerbosity)
could also be retained and directed by the new configuration.
and in this particular case, we need to have this `teeTraceChainTip` to remain here, as this will be hooking into the trace and update the LiveView and output metrics to monitoring.

Reply:
@CodiePP Will do. I'll make the changes.